### DISCUSSION TITLE
Deploy the Argo UI read-only?

### DISCUSSION THREAD

**Discussion by withnale** [argoproj/argo-workflows#6777] [Q&A] [answered]
We'd like to make the Argo UI available via an ingress to various people in our organisation.

However the standard deployment allows for various options such as "Submit New Workflow" via the UI. Is there a way to disable this so that the UI can be used purely to monitoring purposes?

**Comment by alexec** [comment, marked as answer]
Yes. Remove all RBAC write permissions from its role.

**Reply by tooptoop4** [reply]
is there a way to allow submitting from existing workflowtemplates but disable submitting arbitrary new workflow yamls (as they have not gone through code review) ?

**Reply by agilgur5** [reply]
Yes, as I mentioned in #11391, you can use the [Workflow Restrictions](https://argoproj.github.io/argo-workflows/workflow-restrictions/) feature for this.
